library/offcanvas-walker.php

```diff
@@ -2,13 +2,13 @@
 /**
  * Customize the output of menus for Foundation off-canvas menu with multi-level support
  */
-if ( ! class_exists( 'FoundationPress_offcanvas_walker' ) ) :
-class FoundationPress_offcanvas_walker extends Walker_Nav_Menu {
+if ( ! class_exists( 'FoundationPressOffcanvasWalker' ) ) :
+class FoundationPressOffcanvasWalker extends Walker_Nav_Menu {
 
 	function display_element( $element, &$children_elements, $max_depth, $depth = 0, $args, &$output ) {
 		$element->has_children = ! empty( $children_elements[ $element->ID ] );
 		$element->classes[] = ( $element->current || $element->current_item_ancestor ) ? 'active' : '';
-		$element->classes[] = ( $element->has_children && $max_depth !== 1 ) ? 'has-submenu' : '';
+		$element->classes[] = ( 1 !== $element->has_children && $max_depth ) ? 'has-submenu' : '';
```

Comment:
This change seems wrong, previously it was 
`$element->has_children && $max_depth !== 1` 
and now it is 
`1 !== $element->has_children && $max_depth`
shouldn't it rather be
 `$element->has_children && 1 !== $max_depth `
?